reasoning_gym/arithmetic/chain_sum.py

```diff
@@ -108,6 +108,21 @@ def _generate_task(self, rng: random.Random, num_terms: int, min_value: int, max
         expression = " ".join(expression_parts)
         return expression, result
 
+    def score_answer(self, answer: Optional[str], entry: Dict[str, any]) -> float:
+        """Overwrite this method in derived classes if a single oracle answer is not available."""
+        oracle_answer = entry["answer"].strip()
+        reward = 0.0
```

Comment:
It seems the same method is repeatedly used. Should we use a common base class or add a helper function to utils.py?